app/src/main/java/org/sil/storyproducer/model/Workspace.kt

```diff
@@ -168,19 +169,21 @@ object Workspace {
         // Create an App-specific internal or external storage folder for an initial demo
         // https://developer.android.com/training/data-storage
         // https://developer.android.com/training/data-storage/app-specific
-        var appSpecificDocsDir: File?
+        var appSpecificDocsDir: File? = null
         var appSpecificTemplateDir : File? = null
         // Check external memory first then internal memory for enough free space for demo
```

Comment:
As we discussed, could we check first if internal memory has 1GB free, and if so that's what we use. If not, begin searching on external storage for whatever might work best and use that.  Thus the 'normal' install will usually be internal, and it will be easier for Greg and me to know what to expect etc. Thanks!

Reply:
Done